video/out/drm_atomic.c

```diff
@@ -418,6 +445,10 @@ bool drm_atomic_restore_old_state(drmModeAtomicReqPtr request, struct drm_atomic
     // This property was added in kernel 5.0. We will just ignore any errors.
     drm_object_set_property(request, ctx->crtc, "VRR_ENABLED", ctx->old_state.crtc.vrr_enabled);
 
+    // This property is only here if DRM_HDR is enabled, so don't worry about the return
+    // Restores the screen to SDR
+    drm_object_set_property(request, ctx->connector, "HDR_OUTPUT_METADATA", 0);
+
```

Comment:
Unfortunately, this doesn't work, I don't know why. According to https://github.com/mpv-player/mpv/pull/8456#issuecomment-778899198 it should. But it could be a kernel bug.